commands/bot.go

```diff
@@ -236,22 +237,26 @@ func botEnableCmdF(c client.Client, cmd *cobra.Command, args []string) error {
 
 func botDisableCmdF(c client.Client, cmd *cobra.Command, args []string) error {
 	users := getUsersFromUserArgs(c, args)
+
+	var result *multierror.Error
 	for i, user := range users {
 		if user == nil {
 			printer.PrintError(fmt.Sprintf("can't find user '%v'", args[i]))
+			result = multierror.Append(result, fmt.Errorf("can't find user '%v'", args[i]))
```

Comment:
Please use `%q` to quote strings, the original code was lacking in that regard as well. It would be awesome if you could change it there as well.